### PR TITLE
BACKLOG-13313 : remove filemanager and its permission check

### PR DESCRIPTION
**Pull request by Yousria**
## JIRA

https://jira.jahia.org/browse/BACKLOG-13313

### CODE REVIEW

File: src/main/resources/jnt_gotoManager/html/gotoManager.jsp [hunk 1]
```diff
@@ -17,20 +17,6 @@
...
-            <c:set var="multisite" value="true"/>
-        </c:if>
-        <c:if test="${gotoType eq 'content'}">
-            <c:set var="conf" value="editorialcontentmanager"/>
-            <c:set var="requiredPermission" value="editorialContentManager"/>
-            <c:set var="label" value="label.contentmanager"/>
-            <c:set var="icon" value="treepanel-content-manager-1616"/>
-            <c:set var="multisite" value="true"/>
-        </c:if>
         <c:if test="${gotoType eq 'unitedContent'}">
```
Comment (tdraier) [inline src/main/resources/jnt_gotoManager/html/gotoManager.jsp:20]: Maybe you should also update https://github.com/Jahia/default/blob/f794040d553506fff812316584e4f7600989412d/src/main/resources/META-INF/definitions.cnd#L171 ? And eventually remove the property and the if, if there's only one possible value ..

Comment (Yousria) [reply src/main/resources/jnt_gotoManager/html/gotoManager.jsp:20]: yep, I'll fix that

File: src/main/resources/META-INF/definitions.cnd [hunk 2]
```diff
@@ -167,9 +167,6 @@
  - toAdminComponent (string, choicelist[resourceBundle]) = 'all' indexed=no < 'all', 'users', 'groups'
  - buttonClass (string) indexed=no
 
-[jnt:gotoManager] > jnt:content, mix:title, jmix:siteComponent, jmix:studioOnly
```
Comment (tdraier) [inline src/main/resources/META-INF/definitions.cnd:170]: Now you removed the whole component - I don't know if it's a good idea, I don't know where it was used -  but in that case you should remove the jsp .. ?

Comment (Yousria) [reply src/main/resources/META-INF/definitions.cnd:170]: I'll close this PR and start from the beginning

**Pull request closed without merging**